src/aind_metadata_mapper/mesoscope/session.py

```diff
@@ -75,31 +81,18 @@ def __init__(
         else:
             job_settings_model = job_settings
         super().__init__(job_settings=job_settings_model)
-        with open('//allen/programs/mindscope/workgroups/openscope/ahad/medata-mapper/aind-metadata-mapper/tests/resources/open_ephys/camstim_ephys_session.json', 'r') as file:
+        with open(
+            "//allen/programs/mindscope/workgroups/openscope/ahad/medata-mapper/aind-metadata-mapper/tests/resources/open_ephys/camstim_ephys_session.json",
```

Comment:
@Ahad-Allen Can this path get removed? If it is something necessary for stimulus processing then it should be in a resources folder or as a dictionary in stimulus module itself? We should discuss what would be best

Reply:
Yes absolutely! We could create a base json in a resources folder or this repo.